Apps/Database/src/Delegates/DBCommunicationDelegate.cs

```diff
@@ -181,5 +216,51 @@ public DBResult<Communication> Delete(Communication communication, bool commit =
             this.logger.LogDebug($"Finished deleting Communication in DB");
             return result;
         }
+
+        private List<Communication> GetActiveBannerCommunications()
+        {
+            List<Communication> communication = this.dbContext.Communication
+                 .OrderByDescending(c => c.CreatedDateTime)
+                 .Where(c => c.CommunicationTypeCode == CommunicationType.Banner
+                     && c.CommunicationStatusCode == CommunicationStatus.New && DateTime.UtcNow <= c.ExpiryDateTime)
+                 .ToList();
```

Comment:
Is it necessary to bring back a list of Communication objects?  It looks like you want to check if Communications exist.  Is it possible to use Any<> or Count<> instead of ToList()?  Any<> would return a bool and Count would return an int.

Reply:
@BrianMaki  Any() will fail when the list is null and also Count if you use count and list is null .. it will fail